Cargo.toml

```diff
@@ -148,7 +148,8 @@ members = [
     "secure/storage",
     "secure/storage/github",
     "secure/storage/vault",
-    "state-sync",
+    "state-sync/state-sync-v1",
+    "state-sync/state-sync-v2",
```

Comment:
You know, sometimes I wonder why we bother with all the crates if we don't version them, or publish them separately, since we can just do this at any time with no impact.